### PR TITLE
Create getting_started_retrieval.md

### PR DESCRIPTION
**Pull request by lilith-zy**

### CODE REVIEW

File: docs/en/tutorials/getting_started_retrieval.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
+# Quick Start
+---
+At first，please take a reference to [Installation Guide](./install.md)to prepare your environment.
```
Comment (cuicheng01) [inline docs/en/tutorials/getting_started_retrieval.md]: install_en.md

File: docs/en/tutorials/getting_started_retrieval.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+## Content
+
+* [1. Data Preparation](#Data-Preparation)
+* [2. Training and Evaluation on Single GPU](#Training-and-Evaluation-on-Single-GPU)
+  * [2.1 Training](#Training)
+  * [2.2 Resume Training](#Resume-Training)
+  * [2.3 Evaluation](#Evaluation)
+* [3. Export Inference Model](#Export-Inference-Model)
+  
+<a name="Data Preparation"></a>
```
Comment (cuicheng01) [inline docs/en/tutorials/getting_started_retrieval.md]: Data-Preparation,下同

File: docs/en/tutorials/getting_started_retrieval.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+cp {Data storage path}/CUB_200_2011.tgz .
+
+# Unzip
+tar -xzvf CUB_200_2011.tgz
+
+#go to `CUB_200_2011`
+cd CUB_200_2011
+```
+
+When using the dataset for image retrieval task, we usually use the first 100 classes as the training set, and the last 100 classes as the testing set, so we need to process those data so as to adapt the model training of image retrival.
```
Comment (cuicheng01) [inline docs/en/tutorials/getting_started_retrieval.md]: retrieval

File: docs/en/tutorials/getting_started_retrieval.md [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+
+```shell
+#Create train and test directories
+mkdir train && mkdir test
+
+#Divide data into training set with the first 100 classes and testing set with the last 100 classes.
+ls images | awk -F "." '{if(int($1)<101)print "mv images/"$0" train/"int($1)}' | sh
+ls images | awk -F "." '{if(int($1)>100)print "mv images/"$0" test/"int($1)}' | sh
+
+#Generate train_list and test_list
```
Comment (cuicheng01) [inline docs/en/tutorials/getting_started_retrieval.md]: train_list.txt test_list.txt

File: docs/en/tutorials/getting_started_retrieval.md [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+
+
+```
+python3 tools/train.py \
+    -c ./ppcls/configs/quick_start/MobileNetV1_retrieval.yaml \
+    -o Arch.Backbone.pretrained=True \
+    -o Global.device=gpu
+```
+
+ `-c` is used to specify the path to the configuration file, and `-o` is used to specify the parameters that need to be modified or added, where `-o Arch.Backbone.pretrained=True` indicates that the Backbone part uses the pre-trained model, in addition, `Arch.Backbone.pretrained` can also specify backbone.pretrained` can also specify the address of a specific model weight file, which needs to be replaced with the path to your own pre-trained model weight file when using it. `-o Global.device=gpu` indicates that the GPU is used for training. If you want to use a CPU for training, you need to set `Global.device` to `cpu`.
```
Comment (cuicheng01) [inline docs/en/tutorials/getting_started_retrieval.md]: 这里少了一个`

File: docs/en/tutorials/getting_started_retrieval.md [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+        weight: 1.0
+        margin: 0.5
+```
+    
+The final total Loss is a weighted sum of all Losses, where weight defines the weight of a particular Loss in the final total. If you want to replace other Losses, you can also change the Loss field in the configuration file, for the currently supported Losses please refer to [Loss](... /... /... /ppcls/loss).
+
+<a name="Model Recovery Training"></a>
+### 2.2 Model Recovery Training
+
+If the training task is terminated for some reasons, it can be recovered by loading the breakpoint weights file and continue training.
```
Comment (cuicheng01) [inline docs/en/tutorials/getting_started_retrieval.md]: checkpoints

File: docs/en/tutorials/getting_started_retrieval.md [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+
+
+```
+python3 tools/train.py \
+    -c ./ppcls/configs/quick_start/MobileNetV1_retrieval.yaml \
+    -o Global.checkpoints="./output/RecModel/epoch_5" \
+    -o Global.device=gpu
+```
+
+There is no need to modify the configuration file, just set the `Global.checkpoints` parameter when continuing training, indicating the path to the loaded breakpoint weights file, using this parameter will load both the saved breakpoint weights and information about the learning rate, optimizer, etc.
```
Comment (cuicheng01) [inline docs/en/tutorials/getting_started_retrieval.md]: checkpoints

File: docs/en/tutorials/getting_started_retrieval.md [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+
+**Note**：
+
+* The `-o Global.checkpoints` parameter need not contain the suffix name of the breakpoint weights file, the above training command will generate the breakpoint weights file as shown below during training, if you want to continue training from breakpoint `5` then the `Global.checkpoints` parameter just needs to be set to `". /output/RecModel/epoch_5"` and PaddleClas will automatically supplement the suffix name.
```
Comment (cuicheng01) [inline docs/en/tutorials/getting_started_retrieval.md]: checkpoints

File: docs/en/tutorials/getting_started_retrieval.md [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+python3 tools/eval.py \
+    -c ./ppcls/configs/quick_start/MobileNetV1_retrieval.yaml \
+    -o Global.pretrained_model=./output/RecModel/best_model
+```
+
+The above command will use `. /configs/quick_start/MobileNetV1_retrieval.yaml` as a configuration file to evaluate the model obtained from the above training `. /output/RecModel/best_model` for evaluation. You can also set up the evaluation by changing the parameters in the configuration file, or you can update the configuration with the `-o` parameter, as shown above.
+
+Some of the configurable evaluation parameters are introduced as follows.
+* `Arch.name`: the name of the model
+* `Global.pretrained_model`: path to the pre-trained model file of the model to be evaluated, unlike `Global.Backbone.pretrained` where the pre-trained model is the weight of the whole model, whereas `Global.Backbone.pretrained` is only the Backbone.pretrained` is only the weight of the Backbone part. When it is time to do model evaluation, the weights of the whole model need to be loaded.
```
Comment (cuicheng01) [inline docs/en/tutorials/getting_started_retrieval.md]: 少一个`

**Review by cuicheng01 — DISMISSED** [review]

**Review by cuicheng01 — APPROVED** [review]
